package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@kgs-research/utils",
-    "version": "0.0.9",
+    "version": "0.1.0",
```

Comment:
Updating the minor version (0.X.0) since npm defaults to checking this value for package updates.